posts/key-learnings-from-waku-hackers-in-2023

```diff
@@ -0,0 +1,79 @@
+
+Václav acknowledged the limited nature of the current documentation and examples; it is a symptom of Waku’s relative youth and constant evolution. Václav encouraged community participation by highlighting the existence of the[ bounties repository](https://github.com/waku-org/bounties/), where contributors can propose and work on examples aligned with specific frameworks or use cases. This collaborative approach aims to address gaps in documentation and make Waku more accessible to a broader audience.
+
+We are happy to share that since the X Space happened, we have released a [Waku React package](https://docs.waku.org/guides/js-waku/use-waku-react/) based on Federico's feedback that provides components and UI adapters to integrate @waku/sdk into React applications effortlessly.
+
+
+## We are improving connection issues by better node incentivisation
+
+Martin and Florian shared that their team experienced network connection issues several times during the hackathon, noting that the network was likely overloaded due to the significant number of hackathon participants. Aaron shared his appreciation for the feedback: “We must be brutally honest if we are going to fix the things that aren't easy to work with.” 
+
+Running nodes that fully participate in the Waku Relay Network should be encouraged, and [we have been doing so in this Q](https://blog.waku.org/waku-roadmap) by optimising the software and making participation permissionless. With such innovations, the community can benefit from more neutral contributions to the network, hence the backbone of a secure, scalable network, and a more extensive set of nodes to get sender and recipient k-anonymity.
+
+
+## You called for storage… and it is coming!
+
+The hackers mentioned that using Waku alone without any additional decentralised storage solution limits what they can build during a hackathon's time-constrained period. As Václav noted: “Waku is in general ephemeral, and it should not be used, and it cannot be used for long-term storage.”
+
+Until now, hackers have relied on alternatives such as IPFS and Arweave when they need a decentralised storage element for their projects. However, both options rely on some form of centralised gateways to access external data.
+
+The good news is [Codex](https://codex.storage/) will soon be ready to step in and fill the gap. As the storage layer of the [Logos](logos.co) technology stack, Codex is durable and decentralised. Codex implements protocol constructs that incentivise wide participation of data storage providers, both small and large, thus ensuring a robust network resistant to censorship and external attack.
```

Comment:
```suggestion
The good news is [Codex](https://codex.storage/) will soon be ready to step in and fill the gap. As the storage layer of the [Logos](logos.co) technology stack, Codex is durable and decentralised. Codex implements protocol constructs that incentivise wide participation of data storage providers, both small and large, thus ensuring a robust network resistant to censorship and external attack.
```

I would not emphasis on the "soon" but instead that Codex and Waku are part of the Logos technology stack and that we acknowledge end-to-end solution and good integration between these projects will allow us to deliver what developers want.